provider/openstack/provider.go

```diff
@@ -560,16 +585,31 @@ func (e *Environ) PrecheckInstance(series string, cons constraints.Value, placem
 // PrepareForBootstrap is part of the Environ interface.
 func (e *Environ) PrepareForBootstrap(ctx environs.BootstrapContext) error {
 	// Verify credentials.
-	if err := authenticateClient(e.client); err != nil {
+	if err := authenticateClient(e.client()); err != nil {
 		return err
 	}
+	if !e.supportsNeutron() {
+		logger.Warningf(`Using deprecated OpenStack APIs.
+
+  Neutron networking is not supported by this
```

Comment:
I've made it a bit longer, but not to the same length as the standard messages. I want this to stand out, so it is offset from both the start and end of the lines.